src/structures/display/ChoiceEmbed.ts

```diff
@@ -0,0 +1,83 @@
+import { Message } from "eris";
+import { MessageEmbed } from "../client/RichEmbed";
+import { MessageCollector } from "eris-collector";
+import { Zephyr } from "../client/Zephyr";
+import { createMessage } from "../../lib/discord/message/createMessage";
+
+export class ChoiceEmbed {
+  readonly maxEntries = 12;
```

Comment:
hard cap to how many items to display (to prevent hitting the embed char limit and limit spam), I picked 12 pretty arbitrarily